Nelson Mosquera/webpack-exercise/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "webpack-exercise",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "npx webpack",
```

Do 
```suggestion
    "start": "webpack",
```
since webpack is one of your dependencies.